landingai/vision_pipeline.py

```diff
@@ -252,6 +254,36 @@ def show_image(
 
         return self
 
+    def extend(self, frs: "FrameSet") -> "FrameSet":
+        """Add a all the Frames from `frs` into this FrameSet
+
+        Parameters
+        ----------
+        frs : FrameSet
+            Framerset to be added at the end of the current one
+
+        Returns
+        -------
+        FrameSet
+        """
+        self.frames.extend(frs.frames)
+        return self
+
+    def get_class_counts(self, add_id_to_classname: bool = False) -> Dict[str, int]:
```

Comment:
it'd be nice to add API docs for the function, the parameter `add_id_to_classname`, and the output.

it's also worth adding a small test to cover it.

Reply:
Oops, I missed the doc..... added...I also added a test